### PR TITLE
Implement missing conversions

### PR DESCRIPTION
**Pull request by MikeInnes**
That I was too lazy to fix at the time. Need to add some tests though.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -6,7 +6,7 @@ os:
   # - osx
 
 julia:
-  - 1.0
```
Comment (KristofferC) [inline .travis.yml]: Flux claims to support `1.0` so should probably still keep testing on `1.0`.

Comment (MikeInnes) [reply .travis.yml]: Any thoughts on dropping 1.0 support?

The issue is only a small annoying method ambiguity so it's easy enough to fix, anyway.

Comment (nickrobinson251) [reply .travis.yml]: Please keep supporting (and testing on) the long-term support release of Julia -- it'd be hard to argue for using Flux in Production if that's not the case. (And thanks for your work on Flux!)

**Comment by bhvieira** [comment]
Move to Tracker?

**Pull request closed without merging**